meeting/meeting_september_1_2020.md

```diff
@@ -0,0 +1,63 @@
+# September 1st (7 PM CST)/2nd (9 AM JST) Meeting 2020
+
+## Secp256k1 Branch Progress
+
+* schnorrsig to be merged into secp256k1 soon ([#558](https://github.com/bitcoin-core/secp256k1/pull/558)) after most recent [BIP 340 change](https://github.com/sipa/bips/pull/210)
+* ZKP Branch built on top of updated schnorrsig containing ECDSA adaptor sigs - upcoming
+  * ZKP to move to using secp256k1 as a git subtree instead of rebasing all the time
+* [Temporary non-zkp branch with everything DLC's need on it (with stable APIs that will appear on the final zkp branch ... I think)](https://github.com/nkohen/secp256k1/tree/temp-everything)
+  * [Same branch but with JNI integration as well](https://github.com/nkohen/secp256k1/tree/temp-everything-with-jni)
+
+## Oracle Specifications
+
+* [How should we do number decomposition?](https://github.com/discreetlogcontracts/dlcspecs/issues/65)
+* [URI format proposal](https://github.com/discreetlogcontracts/dlcspecs/pull/63)
+* [Descriptor proposal](https://github.com/discreetlogcontracts/dlcspecs/pull/55)
+
+## TLVs
+
+* Size issue
+  * We are going to have messages much larger than allowed in LN
```

Comment:
Just FYI, BOLT 8 "The maximum size of any Lightning message MUST NOT exceed 65535 bytes."